package.json

```diff
@@ -41,94 +43,91 @@
   "scripts": {
     "clean": "rm package-lock.json; rm -rf node_modules",
     "clean:install": "npm run clean && npm install",
+    "clean:build": "rm -rf build",
     "compile": "babel src --out-dir lib --copy-files",
     "danger": "danger ci",
     "jest": "jest --config jestConfig.js",
-    "lint": "eslint --ext .js,.jsx .",
-    "nightwatch": "nightwatch",
-    "nightwatch:port-provided": "nightwatch -c tests/test.nightwatch.conf.js",
+    "lint": "eslint --ext .js .",
     "pack": "webpack --config tests/test.config.js",
-    "postpublish": "git push --tag",
-    "precompile": "rm -rf lib",
     "prepare": "npm run compile",
     "prepublishOnly": "npm whoami && check-installed-dependencies && npm test",
     "pretest": "npm run lint",
     "release:major": "npm version major -m 'Released version %s' && npm publish",
     "release:minor": "npm version minor -m 'Released version %s' && npm publish",
     "release:patch": "npm version patch -m 'Released version %s' && npm publish",
-    "start": "node scripts/serve/serve-cli.js --config ./tests/test.config",
-    "start-static": "node scripts/serve/serve-static-cli.js --config ./tests/test.config",
-    "test": "npm run jest && npm run nightwatch && npm run nightwatch:port-provided && npm run wdio",
-    "wdio": "npm run wdio-webpack-obj && npm run wdio-webpack-func && npm run tt-wdio-webpack-obj && npm run tt-wdio-webpack-func && npm run tt-wdio-unopinionated",
+    "start": "./bin/serve.js --config ./tests/test.config",
+    "start-static": "npm run pack && ./bin/serve-static.js --site ./build",
+    "test": "npm run jest && npm run wdio",
+    "wdio": "npm run clean:build && npm run wdio-webpack-obj && npm run wdio-webpack-func && npm run tt-wdio-webpack-obj && npm run tt-wdio-webpack-func && npm run tt-wdio-unopinionated && npm run tt-wdio-static",
     "wdio-webpack-obj": "wdio --suite opinionated",
     "wdio-webpack-func": "TT_TEST_WDIO_FUNCTION=true wdio --suite opinionated",
     "tt-wdio-webpack-obj": "node scripts/wdio/wdio-runner-cli.js --suite opinionated",
     "tt-wdio-webpack-func": "TT_TEST_WDIO_FUNCTION=true node scripts/wdio/wdio-runner-cli.js --suite opinionated",
-    "tt-wdio-unopinionated": "npm run pack && node scripts/wdio/wdio-runner-cli.js --formFactors=['tiny','huge'] --locales=['en','fr'] --suite unopinionated && rm -rf ./build"
+    "tt-wdio-unopinionated": "TT_TEST_WDIO_FUNCTION=true node scripts/wdio/wdio-runner-cli.js --formFactors=['tiny','huge'] --locales=['en','fr'] --suite unopinionated;",
+    "tt-wdio-static": "npm run pack; wdio --suite static; rm -rf ./build"
   },
   "dependencies": {
     "async": "^2.5.0",
-    "autoprefixer": "^8.5.2",
-    "axe-core": "3.0.3",
-    "babel-loader": "^7.1.2",
+    "autoprefixer": "^9.5.1",
+    "axe-core": "^3.2.2",
+    "babel-loader": "^8.0.5",
     "babel-polyfill": "^6.23.0",
```

Comment:
We should look at upgrading the babel polyfill. In framework I'm getting duplicate re-generator runtimes.
https://www.npmjs.com/package/@babel/polyfill
This polyfill has be depricated and they recommend using core-js and regenerator runtime